packages/react-core/src/demos/JumpLinks.md

```diff
@@ -30,14 +30,32 @@ import {
   SidebarPanel,
   Switch,
   Title,
-  TextContent
+  TextContent,
+  getResizeObserver
 } from '@patternfly/react-core';
 import DashboardWrapper from './examples/DashboardWrapper';
 
 ScrollspyH2 = () => {
   const headings = [1, 2, 3, 4, 5];
 
   const [isVertical, setIsVertical] = React.useState(true);
+  const [offsetHeight, setOffsetHeight] = React.useState(10);
+
+  // Update offset based on the masthead and jump links nav heights.
+  React.useEffect(() => {
+    const masthead = document.getElementsByClassName('pf-c-masthead')[0];
+    const offsetForPadding = 10;
+
+    getResizeObserver(masthead, () => {
+      if (isVertical) {
```

Comment:
The vertical layout moves the sticky jumplinks header to the top on mobile and pushes the page content down below it, so we'll need to check for & accommodate the vertical mobile layout when calculating the offset.

Reply:
@evwilkin, I was hoping you could elaborate on the expected behavior here. I did notice the stacking of the header when in vertical mode and with a small enough window width (< 768px to be exact). As it stands now, when expanding the list and clicking on 1 nav item, the list collapses and the correct `Heading` is shown/scrolled to.

Reply:
@jeffpuzzo I noticed that though the page jumps to the correct spot, the scrollspy functionality isn't tracking the active section correctly (actually both on mobile & desktop).  This is a bug unrelated to these changes that [I've logged](https://github.com/patternfly/patternfly-react/issues/6749), sorry for the confusion!

Reply:
I see what you mean now. Good catch. I'll take a look at your fix.